src/main/java/org/opensearch/knn/plugin/transport/KNNStatsRequest.java

```diff
@@ -23,8 +24,7 @@ public class KNNStatsRequest extends BaseNodesRequest<KNNStatsRequest> {
      * Key indicating all stats should be retrieved
      */
     public static final String ALL_STATS_KEY = "_all";
-
-    private Set<String> validStats;
+    private Set<String> validStats = StatNames.getNames();
```

Comment:
init this variable via constructor rather than here, this will help us mock things later on if required.

Reply:
Will update.